data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -253,17 +256,15 @@
     field_names.LINGUISTIC_ISO_FIELD,
     field_names.LINGUISTIC_ISOLATION_LOW_HS_EDUCATION_FIELD,
     field_names.UNEMPLOYMENT_FIELD,
-    field_names.LINGUISTIC_ISO_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
     field_names.UNEMPLOYMENT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.LINGUISTIC_ISO_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
```

Comment:
Is this right? Why are we separating the linguistic isolation fields from each other? (I'm also happy to merge and sort out small details like this later.)

Reply:
I asked about this and another instance of this in the spreadsheet